utils/towers.ahk

```diff
@@ -29,7 +29,7 @@ Targeting(tower, tabCount) {
     Sleep(200)
 }
 
-Upgrade(tower, topCount, middleCount, bottomCount) {
+Upgrade(tower, topCount, middleCount := 0, bottomCount := 0) {
```

Comment:
Default values for middleCount and bottomCount were left out on purpose to always require all three paths in the upgrade function so it's always consistant

Reply:
You can use the first part of the upgrade to upgrade heroes. In such a case, that's just 2 additional 0's.
Can be reverted though.

Reply:
Well, upgrading the hero is a good enough reason for this change.